server/views/arrivals/_non-arrival_form.njk

```diff
@@ -0,0 +1,48 @@
+<form action="/premises/{{premisesId}}/bookings/{{bookingId}}/nonArrivals" method="post">
+  <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+  {{ govukDateInput({
+    id: "date",
```

Comment:
These IDs are now duplicated on the page, which would result in invalid HTML (and probably some accessiblity issues). Might be better to namespace the IDs with the attribute type (maybe `nonArrival[date]` in this context)

Reply:
a very good point!